lib/search.rb

```diff
@@ -1100,14 +1100,13 @@ def self.ts_query(term: , ts_config:  nil, joiner: nil, weight_filter: nil)
 
   def self.to_tsquery(ts_config: nil, term:, joiner: nil)
     ts_config = ActiveRecord::Base.connection.quote(ts_config) if ts_config
-    tsquery = "TO_TSQUERY(#{ts_config || default_ts_config}, '#{term}')"
+    tsquery = "TO_TSQUERY(#{ts_config || default_ts_config}, $$#{term}$$)"
```

Comment:
This is cool - I did not know about this kind of escaping.

Reply:
O this is an alternate quoting syntax in PG: https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-DOLLAR-QUOTING

Reply:
Apparently we can't....I was actually trying to figure out what to do here and the simplest is just to escape single quotes and backslash manually. 

Actually we can if we do `"$#{random_number}$$test$#{random_number}$"` but I'll prefer if we just escape.